docs/docsgen/source/conf.py

```diff
@@ -12,7 +12,7 @@
 # -- Project information -----------------------------------------------------
 
 author = "ONNX"
-copyright = "2022"
+copyright = "2023"
```

Comment:
nit: can the year be auto-generated?

Reply:
Not sure what it should be, however. Based on date of original or when documentation-generation runs.